docs/middleware.md

```diff
@@ -4,8 +4,8 @@ Next 12 introduces a new feature called [Middleware](https://nextjs.org/docs/mid
 a request has finished processing. Middleware can be used to modify the request or replace the response. For example, it
 can change headers, rewrite the request path, or return a different response entirely.
 
-Next.js Middleware can run either in an edge function (highly recommended for version 12.2+ as ISR will not work otherwise) or at the origin. On Netlify, middleware runs at the origin as
-part of the normal Next.js server.
+Next.js Middleware can run either in an edge function (highly recommended for version 12.2+ as ISR will not work
+otherwise) or at the origin. On Netlify, middleware runs at the origin as part of the normal Next.js server.
```

Comment:
Should this potentially be the following?

```suggestion
otherwise) or at the origin. On Netlify, Middleware runs at the origin as part of the normal Next.js server.
```

I'm unsure here if we're talking about middleware the feature (Next.js Middleware) or the general middleware concept/functionality that applies to any framework